README.md

```diff
@@ -1,13 +1,14 @@
 Cloud Native Application
 ========================
 
-This document purpose is to describe the Cloud Native Application training described below and which is in particular delivered to ENSIMAG IS students in 2016 and 2017.
+This document purpose is to describe the Cloud Native Application training described below and which is in particular delivered to ENSIMAG IS students in 2017 and 2018.
```

Comment:
2 times describe in the original doc was also too much...
What I meant was in 2016-2017 and 2017-2018.